### ISSUE TITLE
know about level sample compare result

### ISSUE THREAD

**Issue by aman21392** [nanoporetech/tombo#276]
I have direct-RNA sample nanopore data one is control and another is treated sample. So I just want to know the m6A modification in my treated sample. I used human_transcript.fasta file for resquiggle command -- 
"tombo resquiggle single_fast5/  homotranscript.fa --overwrite --processes 50"
Then I used 
tombo detect modifications level_sample_compare --fast5-basedirs treated/single_fast5/ --alternate-fast5-basedirs control/single_fast5/ --minimum-test-reads 1 --processes 50 --num-most-significant-stored 14000000 --statistics-file-basename sample.level_samp_comp_detect

Then I used command--
tombo text_output browser_files --fast5-basedirs treated/single_fast5/ --control-fast5-basedirs control/single_fast5/ --statistics-filename sample.level_samp_comp_detect.tombo.stats --browser-file-basename sample.level_samp_comp_detect --file-types statistic --motif-descriptions DRACH:3:dam_6mA
I got 2 wiggle file. plus and minus and I think only plus wig file we mainly use.
I show you my result of plus.wig file looks like---
variableStep chrom=ENST00000203630 span=1
531 0.8333
532 1.0000
533 1.0000
534 1.0000
535 1.0000
536 0.8333
537 0.8333
538 0.8333
539 0.8333
540 0.8333
541 0.8333
542 1.0000
543 1.0000
544 1.0000
545 1.0000
546 1.0000
547 1.0000
548 0.8333
549 0.8333
550 0.8333
551 0.8333
552 0.8333
553 0.6667
554 0.6667
555 0.5000
556 0.5000
557 0.6667
558 0.8333
559 1.0000
560 1.0000
561 1.0000
562 1.0000
563 1.0000
564 1.0000
565 1.0000
566 0.8333
567 0.6667
568 0.5000
569 0.6667
570 0.8333
571 1.0000
572 0.8333
573 0.8333
574 0.8333
575 0.8333
576 0.8333
577 0.6667
578 0.8333
579 0.8333
580 0.8333
581 0.6667
582 0.6667
583 0.8333

I run all command correctly or not .Please just me help to understand which region is m6A modification what is it means of wiggle output and what these column means. 
Thanks in advance

**Comment by marcus1487** [comment]
Yes all commands appear correct. The `tombo detect_modifications level_sample_compare` defaults to the KS-test and stored the effect size statistic by default (for the KS test this is the D-statistic). The larger the value of this statistic the greater likelihood of a modified base in general. See more information about this in the [docs here](https://nanoporetech.github.io/tombo/modified_base_detection.html#canonical-sample-comparison-methods).

I would warn away from setting `--minimum-test-reads 1`. The statistics produced by the sample comparison methods generally require higher coverage in order to be useful (hence the default of 50X minimum coverage). Lower coverage is likely the reason for the large number of positives (likely false positives) in the presented snippet of your results.

As for specific detection of 6mA this is left to the user to determine the identity of the modified base. Tombo provides tools and a framework to identify consistent shifts in raw nanopore signal. The interpretation of these shifts is left to the user.

**Comment by aman21392** [comment]
Thanks for you reply. I using --minimum-test-reads 1 because i think if I have some low coverage of reads than we don't want to miss those reads. I uses RNA sample and i just think may I have some RNA which have lower reads. So i don't want to loose them. If I used --minimum-test-reads 1 than I don't get perfectly modification result. 
What you say. Thanks in advance

**Comment by FerchoHQ** [comment]
Hi, I have a question regarding to "tombo detect_modifications model_sample_compare",  what means --control-fast5-basedirs? how do you get this. 

![image](https://user-images.githubusercontent.com/50809731/153054234-9e85593f-42d1-4a17-83d3-2cdad9000eb2.png)

Thanks for your attention.
